cns/restserver/ipam.go

```diff
@@ -110,6 +135,22 @@ func (service *HTTPRestService) requestIPConfigHandler(w http.ResponseWriter, r
 		return
 	}
 
+	// Check the status of the NC.
+	ncStatuses := multicontainerstatus(service.state.ContainerStatus)
```

Comment:
Update the UT to verify these changes as well

Reply:
There aren't any existing tests that use requestIPConfigHandler